### DISCUSSION TITLE
glassmorphism-generator/

### DISCUSSION THREAD

**Discussion by giscus** [zxuqian/zxuqian.cn#76] [Announcements]
# glassmorphism-generator/

CSS Glassmorphism 玻璃特效生成器，可以通过滑杆调整玻璃的颜色、透明度等，并最终生成直接可用的 CSS 代码。

https://zxuqian.cn/glassmorphism-generator/

**Comment by codingJJJ** [comment]
6

**Comment by Vicwxy9** [comment]
5656
